packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -3868,16 +3869,73 @@ export default class Meeting extends StatelessWebexPlugin {
 
     Metrics.postEvent({event: eventType.MOVE_MEDIA, meeting: this});
 
-    return MeetingUtil.joinMeetingOptions(this, {resourceId, moveToResource: true}).then(() => {
-      this.meetingFiniteStateMachine.join();
+    this.locusInfo.once(LOCUSINFO.EVENTS.SELF_OBSERVING, async () => {
+      // On moveFrom ask the developer to re capture it moveFrom  then updateMedia
```

Comment:
1) yes comments are in a wrong place changed it 
2) yes the existing api wont have any signature change but ya they will have to call updateMedia rather then addMedia because there is a existing share connection which is active so we are just going to update it and get back audio and video 

3) By asking the developer to do updateMedia they have full control on what media they can send currently by default we were enabling audio and video by default 

No one currently have been using  moveFrom feature its has been broken for a long time so we will have to update the doc saying its a breaking change in the node